server/lobbyconnection.py

```diff
@@ -505,14 +516,75 @@ async def check_policy_conformity(self, player_id, uid_hash, session, ignore_res
 
         return response.get("result", "") == "honest"
 
+    async def command_auth(self, message):
+        token = message["token"]
+        unique_id = message["unique_id"]
+        player_id = await self.oauth_service.get_player_id_from_token(token)
+        auth_method = "token"
+
+        async with self._db.acquire() as conn:
+            result = await conn.execute(
+                select([t_login.c.login, t_login.c.steamid])
+                .where(t_login.c.id == player_id)
+            )
+            row = await result.fetchone()
+
+            if not row:
+                self._logger.error("User id from token not found in database possible fraudulent token")
```

Comment:
But make sure it fits on 1 line
```suggestion
                self._logger.warning("User id from token not found in database possible fraudulent token: %s", player_id)
```